src/parser/view_transform.c

```diff
@@ -2622,12 +2626,27 @@ mq_substitute_subquery_in_statement (PARSER_CONTEXT * parser, PT_NODE * statemen
 	      parser_free_tree (parser, tmp_class);
 	    }
 
+	  select_list = query_spec->info.query.q.select.list;
+
+	  /* exclude the first oid attr, append non-exist attrs to select list */
```

Comment:
```suggestion
	  /* exclude the first oid attr */
```
무언가 추가되는 내용이 필요하다면 position을 update 한다고 써야 할 것 같습니다.

Reply:
추가되는 내용이 없습니다. 리뷰해주신 내용처럼 제거하겠습니다.